tests/components/sensorpush_cloud/conftest.py

```diff
@@ -0,0 +1,83 @@
+"""Common fixtures for the SensorPush Cloud tests."""
+
+from __future__ import annotations
+
+from collections.abc import Callable, Generator
+from typing import Any
+from unittest.mock import AsyncMock, patch
+
+import pytest
+from sensorpush_ha import SensorPushCloudApi, SensorPushCloudHelper
+
+from homeassistant.components.sensorpush_cloud.const import DOMAIN
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD
+
+from tests.common import MockConfigEntry
+
+
+def pytest_configure(config: pytest.Config) -> None:
+    """Register markers for tests that use data."""
+    config.addinivalue_line("markers", "data: mark test to run with specified data")
```

Comment:
whats this?

Reply:
It registers a custom marker in pytest. Without it, you end up with a warning:
```
test_config_flow.py:19
  /Users/sstallion/Source/homeassistant-core/tests/components/sensorpush_cloud/test_config_flow.py:19: PytestUnknownMarkWarning: Unknown pytest.mark.data - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html
    @pytest.mark.data(MOCK_DATA)
```
That said, this will be removed once I move MOCK_DATA out of a marker and into a dedicated mock class.